### PR TITLE
Add ZK-3794 test case

### PR DESCRIPTION
**Pull request by klyvechen**

### CODE REVIEW

File: zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/ class B85_ZK_3794Test extends ZTL4ScalaTestCase {
+  @Test
+  def testClick() = {
+    runZTL(
+      () => {
+        var textVal = getEval("jq('.z-textbox').val()").trim()
+        var labelTxt = getEval("jq('.z-label:eq(1)').text()").trim()
+        waitResponse()
```
Comment (DevChu) [inline zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala]: why getEval

Comment (klyvechen) [reply zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala]: ok

File: zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/ class B85_ZK_3794Test extends ZTL4ScalaTestCase {
+  @Test
+  def testClick() = {
+    runZTL(
+      () => {
+        var textVal = jq(".z-textbox").`val`().trim;
+        var labelTxt = jq(".z-label:eq(1)").text().trim();
+        waitResponse()
```
Comment (scribetw) [inline zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala]: No need to `waitResponse()` here (No AJAX or animations)

Comment (klyvechen) [reply zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala]: ok

**Pull request merged**